### DISCUSSION TITLE
It said "No such file or directory"

### DISCUSSION THREAD

**Discussion by HarikoNasume** [mishakorzik/AllHackingTools#74] [Q&A] [answered]
![Screenshot_20230823_151339](https://github.com/mishakorzik/AllHackingTools/assets/142973622/bd5bd7bb-2772-40cc-b514-51645ab9215c)

**Comment by mishakorzik** [comment, marked as answer]
Copy and paste this command: cd && cd AllHackingTools && bash Install.sh
